### PR TITLE
Update DevExtreme and DevExpress ASP.NET to 16.1.8

### PR DESCRIPTION
**Pull request by BingoRUS**

### CODE REVIEW

**Comment by yuit** [comment]
@BingoRUS  Thank you for the PR. Just wonder why this is such a large diff? Could you summarize what your change and provide some document?

**Comment by BingoRUS** [comment]
Hey @yuit!

We haven't updated DevExtreme TypeScript Definitions a couple of months already so I decided to add not only one for the latest version but some for previous versions too (see devextreme-XX.X.X.d.ts files). People can use previous versions and they will find these files useful.

The file devextreme.d.ts (valid for DevExtreme 16.1.7) has large diff as well because the new major version of DevExtreme (16.1) contains [a lot of new features](https://js.devexpress.com/New/16_1/).

**Comment by BingoRUS** [comment]
Hi guys, I've updated the pull request with TypeScript Definitions to yesterday released DevExtreme v16.1.8. Please review the changes.

**Comment by romanresh** [comment]
Hello, I've also updated the DevExpress ASP.NET/MVC definitions (devexpress-web) to v16.1.8. 

See: https://www.devexpress.com/Support/Center/VersionHistory?TechnologyName=.NET&PlatformName=ASP.NET+Web+Forms&ProductName=AllProducts&VersionHistoryItemsType=&StartBuildName=16.1.8&EndBuildName=16.1.8&version-mode=single-mode
